authorization/README.md

```diff
@@ -27,11 +27,12 @@ Here is a sample bash script to generate the provided authorizing wiring if you
 ```bash
 ./gradlew :authorization-codegen:shadowJar
 
+DEEPHAVEN_VERSION="$(./gradlew printVersion -q)"
```

Comment:
If a branch has just been checked out, the stdout of the  grade script is not just the version, it's also some other output.

Reply:
Interesting; I would have thought that goes to stderr, but I can see this as a potential issue. There is _probably_ some way to disable that if it's true. That said, the previous line `./gradlew :authorization-codegen:shadowJar` _might_ take care of any first-time startup stdout logging?

Reply:
Ah, good point.